www/main/addons/secondary-window/Markdown.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from 'react';
+import ReactMarkdown from 'react-markdown';
```

Comment:
Do we need both `ReactMarkdown` and `marked`?

Reply:
marked is to properly read the file (tables dont get parsed properly). React markdown is to render the markdown